styles.css

```diff
@@ -0,0 +1,735 @@
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+}
+
+#speakers .person {
+  display: flex;
+  gap: 1rem;
+}
+
+#speakers .person.mobile-hide {
+  display: none;
+}
+
+#speakers .person-graphic {
+  position: relative;
+}
+
+#speakers .person-graphic::before {
+  position: absolute;
+  top: -0.5rem;
+  inset-inline-start: -0.5rem;
+  width: calc(6 * 8px + 4px);
+  height: calc(6 * 8px + 4px);
+  content: '';
+  z-index: -1;
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='8' height='8' viewBox='0 0 8 8'%3E%3Cg fill='%23000000' fill-opacity='.2'%3E%3Cpath fill-rule='evenodd' d='M0 0h4v4H0V0zm4 4h4v4H4V4z'/%3E%3C/g%3E%3C/svg%3E");
+}
+
+#speakers .person img {
+  width: 7rem;
+  height: 7rem;
+  background-color: var(--secondary);
+  flex-shrink: 0;
+}
+
+#speakers .person-info {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+#speakers .person-name {
+  font-size: 1.3rem;
+  font-weight: 500;
+}
+
+#speakers .person-title {
+  color: var(--secondary-dark);
+  font-weight: 700;
+  padding-bottom: 0.75rem;
+  position: relative;
+}
+
+#speakers .person-title::after {
+  border-bottom: 2px solid var(--nutral);
+  position: absolute;
+  height: 1px;
+  width: 1rem;
+  content: '';
+  bottom: 0;
+  inset-inline-start: 0;
+}
+
+#speakers .person-bio {
+  color: var(--primary);
+  opacity: 0.8;
+  font-size: 0.9rem;
+}
+
+#speakers .more {
+  border: 1px solid var(--nutral);
+  border-radius: 0.25rem;
+  padding: 1rem;
+  display: flex;
+  justify-content: center;
+  text-transform: uppercase;
+  gap: 0.5rem;
+}
+
+#speakers .more i {
+  height: 1.5rem;
+  color: var(--secondary);
+}
+
+#sponsors {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  background-color: #333;
+  color: var(--nutral);
+  padding: 2rem 1rem;
+}
+
+#sponsors .section-heading {
+  color: var(--nutral);
+}
+
+#sponsors .sponsors-list {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  align-content: flex-start;
+  gap: 2rem;
+  margin-top: 2rem;
+}
+
+#sponsors .sponsors-list img {
+  opacity: 0.25;
+  transition: opacity 300ms;
+}
+
+#sponsors .sponsors-list img:hover {
+  opacity: 0.75;
+}
+
+footer {
+  padding: 3rem 2rem;
+}
+
+footer .wrapper.wrapper {
+  display: flex;
+  align-items: center;
+  gap: 2rem;
+}
+
+footer .branding {
+  color: var(--primary);
+  font-family: var(--sans-font);
+  font-weight: 800;
+  font-size: 1.25rem;
+  line-height: 1em;
+  flex-shrink: 0;
+  flex-basis: 7rem;
+}
+
+.standout {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  text-align: center;
+}
+
+.standout .wrapper.wrapper {
+  display: flex;
+  align-items: center;
+  max-width: 40rem;
+  gap: 2rem;
+  padding-inline: 1rem;
+}
+
+.standout .wrapper .icon {
+  width: 94%;
+  height: 50%;
+  border: 2px solid var(--neutral);
+  padding: 1rem 1rem;
+  border-radius: 0.25rem;
+  transform: scale(0.5);
+  object-position: center center;
+}
```

Comment:
- [ ] Kindly reduce the padding-top or margin-top and padding-bottom or margin-bottom of your images 🤔 I think this is because of the ` transform: scale(0.5);` property used, what if you use width and height properties instead?

![befor](https://user-images.githubusercontent.com/79058364/170043587-68a1ee76-f4a4-4e07-8f3f-23db5b19a7ad.png)